### ISSUE TITLE
SEB doesn't work with NVDA, a screen reader

### ISSUE THREAD

**Issue by mohdshara** [SafeExamBrowser/seb-win#17]
When NVDA is running, safe browser fails to start with this error:
"SetThreadDesktop failed! Looks like the thread has hooks or windows in
the current desktop."

SEB version 2.1.6
windows 10 creator's update
NVDA 2017.2

This is very important because million of blind users rely on a screen reader to do their exams. We currently have to prepare the exam in an alternative method because of this bug.

**Comment by danschlet** [comment]
Are you sure you are using SEB 2.1.6? The source code of SEB 2.1.6 does never display this error string, I just checked confirmed that in the code.

You can observe the splash screen when starting SEB, is it really showing 2.1.6 as the version? Or go into the Program Files  (x86)\SafeExamBrowser\ directory and check if SafeExamBrowser.exe really has version 2.1.6 (you can hover over the file or show properties/details.

And to be sure uninstall SEB using the Windows Control Panel, then check if there are still files in the  directory 
                    
C:\Program Files (x86)\SafeExamBrowser\

If there are some, delete them manually. Then install SEB 2.1.6 again.

**Comment by danschlet** [comment]
And make sure you're using the [release version of SEB 2.1.6](https://github.com/SafeExamBrowser/seb-win/releases/download/v2.1.6/SafeExamBrowserInstaller.exe), not one of the testing builds!

**Comment by danschlet** [comment]
I just tried to install NVDA and SEB 2.1.6 on Windows 10 and for me it works perfectly. If you still have problems (see above), then please provide more information so we can reproduce the problem (and the [SEB log files](http://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane)).

**Comment by mohdshara** [comment]
you're right, I had to go in and delete files manually. now everything works great with 2.16

thanks.

**Comment by danschlet** [comment]
Great, I'm happy we figured it out. I suspect that the installer didn't update your installation properly. I will investigate this. Windows Installer is driving me nuts, I had other update issues before and this problem doesn't seem easy to solve.
